### ISSUE TITLE
Burning hydrogen should create water

### ISSUE THREAD

**Issue by L5108313** [Eggs-D-Studios/wos-issues#349]
### Guidelines

- [X] I checked for duplicate suggestions
- [X] My suggestion intends to add or improve something

### Version

Main (Production)

### Topic

A game mechanic (e.g. a part, a feature, etc)

### Why are you making your suggestion?

This is what realistically should happen. More importantly though, i believe wos needs more funny interactions like this that are both fun and potentially useful.

### What do you propose? What should change, or what should be added?

When a hydrogen gas part burns, it should be replaced with a water part. When this happens in a furnace, the furnace should output water when tank space is available. If tank space is not available, the water should be voided.

**Comment by HvyEarly** [comment]
running a current through water should create hydrogen

**Comment by TheCrazyInsanity** [comment]
No, it should make steam that's already hot

**Comment by HvyEarly** [comment]
> No, it should make steam that's already hot

you're totally right! But that might cause balance issues because of the extra power you get from steam.
